test/WebSites/MvcTagHelpersWebSite/wwwroot/site.js

```diff
@@ -0,0 +1 @@
+﻿
```

Comment:
I'd prefer the visual indication so the functional test is easier to debug/visualize in the case that something breaks, also makes it more consistent with the LinkTagHelper tests.